### ISSUE TITLE
metrics.pytorch.org umbrella issue

### ISSUE THREAD

**Issue by driazati** [pytorch/test-infra#69]
- [ ] investigate if we can reduce deployment complexity by consolidating on Terraform + AWS instead of (terraform -> ansible -> docker swarm -> nginx -> grafana)
- [ ] move logging from json-file to [cloudwatch](https://docs.docker.com/config/containers/logging/awslogs/)
- [ ] add auth via GitHub logins gated on the PyTorch org

**Comment by zhouzhuojie** [comment]
@driazati for the first two (simplify deployment and logging support), I'd vote for app runner (#78) since it's supported out of the box

Some pros
- Managed services
- Scalable solution
- Logging integration
- Isolated from existing VPC networking
- Deployment is simplified as push a docker image, and can decouple the instances access management

Cons
- Public by default (might be ok, since the website is supposed to be public)
- Adding new services means new app runner service. This is due to the strict deployment model of app runner (one Service per Dockerfile per port)
- Vendor lock in (though only for the infra related setup, the docker image and the grafana configuration is still portable)

**Comment by clee2000** [comment]
Metrics page got moved to be away from grafana, closing as no longer relevant?
